File: PPA/DailyFlash/August/19August/Program1.c

#include<stdio.h>

void main() {
	
	char ch;
	printf("Enter a character: ");
	scanf("%c",&ch);

	if(ch>='A' && ch<='Z')
		printf("Letter %c is in Uppercase.\n",ch);
	else if(ch>='a' && ch<='z')
		printf("Letter %c is in Lowercase.\n",ch);
	else
		printf("U did not entered a character\n");
}

File: Java/DailyFlash/August/19Aug/Program1.java

import java.util.Scanner;
class Character {

	public static void main(String[] args) {
	
		Scanner sc =new Scanner(System.in);
		
		char ch;
		System.out.println("Enter a character: ");
		ch=sc.nextLine().charAt(0);
	
		if(ch>='A' && ch<='Z')
			System.out.println("Letter "+ch+" is in Uppercase.");
		else if(ch>='a' && ch<='z')
			System.out.println("Letter "+ch+" is in Lowercase.");
		else
			System.out.println("U enetered a wrong char!");
	}
}

